main.nf

```diff
@@ -677,7 +677,7 @@ process MapReads {
         ${convertToFastq}
         bwa mem -K 100000000 -R \"${readGroup}\" ${extra} -t ${task.cpus} -M ${fasta} \
         ${input} | \
-        samtools sort --threads ${task.cpus} -m 2G - > ${idSample}_${idRun}.bam
+        samtools sort --threads 8 - > ${idSample}_${idRun}.bam
```

Comment:
Remind me why we're requesting 8 here?
won't there be an issue on smaller settings like travis/github actions?

Reply:
It was mostly to mimic @drpatelh label `cpus_8`

Reply:
We can perhaps think of a different way of doing it in the production code

Reply:
Let's see if Travis is failing or not, before fixing it ;-)